docs/config/lua/keyassignment/SetWindowLevel.md

```diff
@@ -0,0 +1,35 @@
+# `SetWindowLevel`
+
+{{since('nightly')}}
+
+Set window level specified by the argument value. eg: `AlwaysOnTop` keeps the current window on top of other windows.
+
+Accepted values: 
+
+ * `"AlwaysOnBottom"`
+ * `"Normal"` (this is the default)
+ * `"AlwaysOnTop"`
+
+```lua
+config.keys = {
+    {
+        key = '[',
+        mods = 'CMD',
+        action = wezterm.action.SetWindowLevel("AlwaysOnBottom")
+    }, 
+    {
+        key = '0',
+        mods = 'CMD|SHIFT',
+        action = wezterm.action.SetWindowLevel("Normal")
+    }, 
+    {
+        key = ']',
+        mods = 'CMD',
+        action = wezterm.action.SetWindowLevel("AlwaysOnTop")
+    }, 
+}
+```
+
+> Note: 
+> This functionality is currently only implemented on macOS. 
+> The assigned values for window level will have no effect on other operating systems.
```

Comment:
Let's use the mkdocs admonition syntax for these, and move it up so that it is just below the `since` macro call at the top.

```suggestion
!!! note 
    This functionality is currently only implemented on macOS. 
    The assigned values for window level will have no effect on other operating systems.
```